draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -568,7 +568,13 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using a JSON-only
+payloads and secured by SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]. Other specifications exist that aim
+to have similar scope. For example, W3C Verifiable Credential Data Model (VCDM) 2.0 [@W3C.VCDM] defines
+a data model for verifiable credentials encoded as JSON-LD, ISO/IEC 18013-5:2021 [@ISO.18013-5] defines a representation
+of verifiable credentials in the mobile document (mdoc) format encoded as CBOR and secured using COSE. Each
+specification can be independently used while a mapping between different data model representations might be possible.
```

Comment:
As discussed on our call, I'm not sure how this helps the reader. Theoretically I can map between all tree structures. But what could be the purpose of that excercise and why do we need to mention it here?

Reply:
I updated the text.